File: php/4092 - L - Ligero.php
<?php
$stdin = fopen('php://stdin', 'r');
$input = fgets($stdin, 10);

list($n) = split(" ",$input);

 if ($n<2) {
	echo "WRONG ANSWER \n";
} else if ($n>3) {
	echo "RUNTIME ERROR \n";
} else {
	echo "ACCEPTED \n";
}

fclose($stdin);
?>

File: Java/4092 - L - Ligero.java
import java.util.*;

public class COJ2 {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int b = sc.nextInt();
        
        if (b<2) {
			System.out.println("WRONG ANSWER");
		} else if (b>3) {
			System.out.println("RUNTIME ERROR");
		} else {
			System.out.println("ACCEPTED");
		}
    }
}